backup/src/main/java/com/instaclustr/backup/CommonBackupArguments.java

```diff
@@ -0,0 +1,115 @@
+package com.instaclustr.backup;
```

Comment:
Whats the dif between `CommonBackupArguments`, `BaseBackupArguments` and `BackupArguments`?

Reply:
This is a relic from our MS backup component that also supports Commitlog archiving so it has a different set of args for the commitlog entry point. I've left it this way as "at some point" we'll re-introduce commitlog archiving, but it can easily be cleaned up